src/openms/source/FORMAT/HANDLERS/MzMLSpectrumDecoder.cpp

```diff
@@ -547,9 +547,11 @@ namespace OpenMS
   std::string MzMLSpectrumDecoder::domParseString_(const std::string& in, std::vector<BinaryData>& data)
   {
     // PRECONDITON is below (since we first need to do XML parsing before validating)
-    static const XMLCh* default_array_length_tag = xercesc::XMLString::transcode("defaultArrayLength");
-    static const XMLCh* id_tag = xercesc::XMLString::transcode("id");
-    static const XMLCh* binary_data_array_tag = xercesc::XMLString::transcode("binaryDataArray");
+    static constexpr XMLCh id_tag[] = {'i','d', 0};
+    static constexpr XMLCh default_array_length_tag[] = { 'd','e','f','a','u','l','t','A','r','r','a','y','L','e','n','g','t','h', 0};
```

Comment:
can you add a comment why it is written like this? e.g., initializer list of uint16?